### ISSUE TITLE
Componentize Global Styles

### ISSUE THREAD

**Issue by jordanlesich** [HausDAO/daohaus-rage#48]
## Details
Currently we're using a globalStyles.css file to handle the app's globals. The DX for this pattern isn't great. Would be much easier to import a component that can be easily added to the app's index.tsx than it would be to serve and import the CSS. Currently we're just copying the CSS, which will not work. 

## Resource

https://frontendmasters.com/courses/design-systems/adding-global-styles/

## Steps
- [x] Use the pattern in the video link to turn our globalStyles.css into a <GlobalStyles /> component that can be imported into any of our apps . 
- [x] Implement Emma's (instructor) style defaults where applicable. 
- [x] Import 'polished' and implement the normalize() pattern from the video, check for any style differences in storybook. 
- [x] Test with one of the Rage apps

**Comment by earth2travis** [comment]
Believe this is a duplicate of: https://github.com/HausDAO/daohaus-monorepo/issues/35

**Comment by jordanlesich** [comment]
This is different. Same name, but two different things unfortunately.

What I'm writing about is a React Component that handles an app's globalStyles. Usually things that an index.css does. What Jon is talking about is the global styles for the component library, which has a lot more to do with theming and tokens.

**Comment by jonathanprozzi** [comment]
Thanks for the clarification Jord! At a glance I thought it was going to be the same as well -- my bad!

I've approached this in the past by wrapping the app with this type of component. I'll review that section of the course as well. Are you thinking that it would be imported on a page by page basis or at a high-level Layout to wrap the app?

**Comment by jordanlesich** [comment]
High level. In the past I've done the wrapping thing as well. The course just sort of plunked the component in the return of Index.tsx. Either probably works the exact same way. I think wrapper is better too. 

I think the real benefit of this would be to explore the default styles with some of the default CSS rules mentioned in the course + some normalization.

In the future, it might be really cool to look at what it would take to be able to import our macros into an app with completely different globalStyles and still render our UI consistently. My first thought was that we could use styles wrapper, just at the child  component level. But that would depend on how styled-components' hashing/scoping system works. Would it only edit body/html elements for items in that wrapper's scope but not above? 

Might be another item worth looking into.

**Comment by earth2travis** [comment]
Cannot proceed with this issue until https://github.com/HausDAO/daohaus-monorepo/issues/35 is complete

**Comment by jonathanprozzi** [comment]
Closed by https://github.com/HausDAO/daohaus-monorepo/pull/55
